src/parameterizations/vertical/MOM_set_viscosity.F90

```diff
@@ -2967,6 +2980,7 @@ subroutine set_visc_init(Time, G, GV, US, param_file, diag, visc, CS, restart_CS
       call get_param(param_file, mdl, "TIDEAMP_VARNAME", tideamp_var, &
                    "The name of the tidal amplitude variable in the input file.", &
                    default="tideamp")
+      CS%drag_bg_vel = 1.e30
```

Comment:
Please add a comment to this line explaining that this large value should never be used and hence it is deliberately breaking dimensional consistency testing, so that we do not need to refer to its description in the PR every time it gets flagged as a hard-coded dimensional constant to know that it should be ignored.